ttnn/cpp/ttnn/operations/conv/conv_transpose2d/conv_transpose2d.cpp

```diff
@@ -142,6 +142,24 @@ Result conv_transpose2d(
 
         log_debug(LogOp, "Padding : ({},{}) ({},{})", input_pad_top, input_pad_bottom, input_pad_left, input_pad_right);
```

Comment:
this block of code from line 124 to line 143 seems to be unused?

Reply:
It's just there for debugging.

Reply:
If it's just for debug I would factor it out into some function